portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,19 +32,36 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-    private List<String> comment = new ArrayList<>();
 
     @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-      String json = convertToJson(comment);
+      Query query = new Query("Task").addSort("comment", SortDirection.DESCENDING);
```

Comment:
is it possible to use a time stamp to sort these? instead of comment. As the comment is text and cannot really be used for sorting?